crates/cxx-qt-lib/src/core/qset/qset_quuid.rs

```diff
@@ -0,0 +1,70 @@
+// SPDX-FileCopyrightText: 2022 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// SPDX-FileContributor: Andrew Hayzen <andrew.hayzen@kdab.com>
```

Comment:
ditto, you here please :)

Reply:
Autogenerated from https://github.com/KDAB/cxx-qt/blob/main/crates/cxx-qt-lib/src/core/qset/generate.sh#L16